plugins/block-dynamic-connection/test/index.ts

```diff
@@ -85,6 +85,11 @@ document.addEventListener('DOMContentLoaded', function () {
 
   const defaultOptions: Blockly.BlocklyOptions = {
     toolbox,
+    plugins: {
+      connectionPreviewer: BlockDynamicConnection.DynamicConnectionPreviewer(
```

Comment:
Update the `usage` section of the readme to say the dev needs to add this stanza to the `plugins` section of their `options` struct.